packages/core/src/components/RadioButton/RadioButton.tsx

```diff
@@ -3,24 +3,25 @@ import { StyleProp, ViewStyle } from "react-native";
 
```

Comment:
It's not clear to me what purpose the plain RadioButton component serves - I think we should make it work inside the group (what this PR does), or just remove it so folks don't get confused as easily.